cmake/Modules/CppUTestGlobalFlags.cmake

```diff
@@ -108,7 +85,7 @@ set(GMOCK_HOME $ENV{GMOCK_HOME})
 if (DEFINED ENV{GMOCK_HOME})
     # GMock pulls in gtest.
     set(CPPUTEST_INCLUDE_GTEST_TESTS 1)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DGTEST_USE_OWN_TR1_TUPLE=1")
+    set(CPPUTEST_CXX_FLAGS "${CPPUTEST_CXX_FLAGS} -DGTEST_USE_OWN_TR1_TUPLE=1")
```

Comment:
Suppoort for `GTEST_USE_OWN_TR1_TUPLE` has been removed from GTest some years ago. Should we consider removing it (does not harm though)? @basvodde

Reply:
https://github.com/google/googletest/commit/7d3b73c85a42811309eac26e5cbe054c40b64785

Reply:
Right now, I don't think CppUtest actually supports the latest version of GTest/GMock. The ones that are supported are also in the travis build and I think we want to keep supporting these. Thus then the define is still needed.